code/game/objects/items/weapons/power_cells.dm

```diff
@@ -95,7 +95,7 @@
 	name = "infinite-capacity power cell!"
 	icon_state = "icell"
 	origin_tech =  null
-	maxcharge = 30000
+	maxcharge = 1000000000 //Good luck charging this, buff it if it gets fully charged
```

Comment:
@ComicIronic that's a custom define, it's just a big number, smaller than PJB's ```1.#INF```

really though the ```use()``` or ```use_power()``` or however functions just returning ```TRUE``` is proper "infinity" power.